styles/design-settings/carnival/parts/_table-settings.scss

```diff
@@ -200,6 +200,15 @@
     )
 ));
 
+@include add_settings((
+  AlternateShadingTable: (
+        _selectors: ('.alternate-shading'),
+        AlternateShadingTableRow: (
+          background-color: enum('ValueSet:::REQUIRED'),
+        ),
+  )
+));
+
```

Comment:
Since each book could have different shading color and this is the only one property that we are defining in settings, we can remove this code totally, because you already added those settings to book.scss.

We can also leave this here in `table-settings.scss` just for the reference. It might be helpful in the future when another book will use alternate shading. Developer will see that they need to specify some things. But because we don't have a general color that we can use, you can add just comment, like here:

![image](https://user-images.githubusercontent.com/40228854/225895396-3aa13075-e366-4cdf-9976-d94bbed0dd4f.png)